core/domain/exp_jobs_one_off.py

```diff
@@ -542,3 +543,75 @@ def reduce(key, values):
         # Combine all values from multiple lists into a single list
         # for that error type.
         yield (key, list(set().union(*final_values)))
+
+
+class ExplorationMigrationValidationJob(jobs.BaseMapReduceOneOffJobManager):
```

Comment:
Should have unit test that verifies at least happy-path behavior.

Reply:
Done!